tests/mcuboot/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = hello_mcuboot
+
+BOARD ?= nrf52dk
+
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := nrf52dk
+
+export IMAGE_VERSION = 1.1.1+1
```

Comment:
> isn't the image always aligned on a flash page boundary?

Actually they define "alignment" not for the pages but for the minimum size they can write on the flash, maybe @utzig knows more about this, but afaik it can change. If we want to align to our flash page boundary it would be `FLASHPAGE_SIZE` which is a lot, and can't be read by imgtool.

> isn't the IMG_HDR always 512 for mcuboot?

Not always afaik, and can be changed as needed. In mynewt is actually 32B, since they relocate the VTOR on RAM so they don't need a 512B alignment as it is needed if we locate the VTOR in ROM. However in RIOT I don't know if we aim to locate VTOR in RAM one day...